scripts/restore/README.md

```diff
@@ -48,6 +48,7 @@ While restoring we need to stop all harbor pods except for the database.
 Create the job and wait until it has completed:
 
 ```
+./bin/ck8s ops kubectl sc apply -n harbor -f scripts/restore/network-policies.yaml
```

Comment:
The other scripts in the folder have harbor in the name.
```suggestion
./bin/ck8s ops kubectl sc apply -n harbor -f scripts/restore/network-policies-harbor.yaml
```

Reply:
Also, will this work for external databases as well? Maybe this whole restore section does not work for that.

Reply:
Harbor restore does not work for external db.

Reply:
Ok, then you could add a note about that if you want (even though that is slightly out of scope).